solutions/authentication/index.asciidoc

```diff
@@ -0,0 +1,54 @@
+= Authentication
+
+Access control is an important aspect for the security in IT application landscapes. There are two different aspects to distinguish:
+
+* Authentication (Who tries to access?)
+* Authorization (Is the one accessing allowed to do what he wants to do?)
+
+This part deals with the recommendations on authentication.
+
+== You have the following problem to be solved
+In large IT landscapes it is a highly recommended best-practice to centralize your authentication. Implemeting the actual authentication into every application or service is therefore considered as an anti-pattern. Instead we suggest to use a central identity and access management (IAM) solution based on established products (e.g. https://www.keycloak.org/[Keycloak]).
+
+=== Using a central IAM
+When using a central IAM, the user is redirected to the identity provider (IdP) when trying to access the application. The IdP returns a login page where the user can log in. After confirming that the user can access the application, the IdP returns a https://jwt.io/[JSON Web Token] (JWT) that the user can use to access the application. Participants integrate with the identity provider using protocols such as https://openid.net/connect/[OpenId Connect], https://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html[SAMLv2] or https://webauthn.io/[WebAuthn]. The original incoming request is forwarded to the actual service and the JWT is added as a bearer token via HTTP header according to https://oauth.net/[OAuth] standard. In the application, the token can be validated according to the user's roles and groups.
```

Comment:
It doesn't have to be a JWT. It can be an arbitrary token but we are suggesting/recommending using JWT

Reply:
Which protocol are we suggesting? OpenId Connect?